app/src/main/java/com/github/se/travelpouch/ui/travel/ParticipantColumn.kt

```diff
@@ -23,36 +19,52 @@ import com.github.se.travelpouch.model.travels.TravelContainer
 import com.github.se.travelpouch.model.travels.fsUid
 
 @Composable
-fun ParticipantRow(
+fun ParticipantColumn(
     participant: Map.Entry<fsUid, Profile>,
     selectedTravel: TravelContainer,
     onClick: () -> Unit
 ) {
 
-  Row(
-      modifier = Modifier.fillMaxWidth().testTag("participantRow").clickable { onClick() },
-      horizontalArrangement = Arrangement.SpaceBetween,
-      verticalAlignment = Alignment.CenterVertically) {
+  Column(
+      modifier =
+          Modifier.fillMaxWidth()
+              .testTag("participantColumn")
+              .clickable { onClick() }
+              .padding(10.dp) // Apply padding for the entire column
+      ) {
+        // Icon on top
         Icon(
             imageVector = Icons.Default.Person,
             contentDescription = "Localized description",
-            modifier = Modifier.padding(5.dp).testTag("participantIcon"))
+            modifier =
+                Modifier.padding(bottom = 8.dp) // Add space below the icon
+                    .testTag("participantIcon"))
+
+        // Username
         Text(
             text = participant.value.name,
-            style = MaterialTheme.typography.titleSmall,
-            fontWeight = FontWeight.Normal,
-            modifier = Modifier.padding(5.dp).testTag("participantName").weight(0.8f))
-        TruncatedText(
+            fontWeight = FontWeight.Bold,
+            modifier =
+                Modifier.fillMaxWidth()
+                    .padding(bottom = 4.dp) // Add space between text elements
+                    .testTag("participantName"))
+
+        // Email
+        Text(
             text = participant.value.email,
-            fontWeight = FontWeight.Light,
-            style = MaterialTheme.typography.labelSmall,
-            maxLength = 25,
-            modifier = Modifier.padding(5.dp).testTag("participantEmail").weight(0.8f))
-        TruncatedText(
+            fontWeight = FontWeight.Bold,
+            modifier =
+                Modifier.fillMaxWidth()
+                    .padding(bottom = 4.dp) // Add space between text elements
+                    .testTag("participantEmail"))
+
+        // Role
+        Text(
             text = selectedTravel.allParticipants[Participant(participant.key)]!!.name,
             fontWeight = FontWeight.Bold,
-            style = MaterialTheme.typography.labelMedium,
-            maxLength = 20,
-            modifier = Modifier.padding(5.dp).testTag("participantRole").weight(1f))
+            modifier =
+                Modifier.fillMaxWidth()
+                    .padding(bottom = 4.dp) // Add space between text elements
+                    .testTag("participantRole"))
```

Comment:
why did you omit the style and leave it as default? If you want the same style (fontsize, font etc) you can just set it to each field so if we change the font, or fontsize it will reflect uniformly